app/src/main/java/io/neurolab/fragments/StatisticsFragment.java

```diff
@@ -0,0 +1,123 @@
+import android.widget.Toast;
+
+import io.neurolab.R;
+import io.neurolab.model.DefaultFFTData;
+import io.neurolab.program_modes.MemoryGraphParent;
+
+public class StatisticsFragment extends Fragment {
+
+    private TextView deltaValView;
+    private TextView thetaValView;
+    private TextView lowAlphaValView;
+    private TextView highAlphaValView;
+    private ImageView concEmojiView;
+    private ImageView sleepEmojiView;
+    private ImageView calmEmojiView;
+    private ImageView angerEmojiView;
+
+    public static String[] parsedData;
+
+    public static StatisticsFragment newInstance() {
+        StatisticsFragment fragment = new StatisticsFragment();
+        return fragment;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        ((MemoryGraphParent) getActivity()).setActionBarTitle(getResources().getString(R.string.statistics));
+        View rootView = inflater.inflate(R.layout.fragment_statistics, container, false);
+        deltaValView = rootView.findViewById(R.id.delta_value);
+        thetaValView = rootView.findViewById(R.id.theta_value);
+        lowAlphaValView = rootView.findViewById(R.id.low_alpha_value);
+        highAlphaValView = rootView.findViewById(R.id.high_alpha_value);
+
+        concEmojiView = rootView.findViewById(R.id.concentrate_emoji);
+        sleepEmojiView = rootView.findViewById(R.id.sleep_emoji);
+        calmEmojiView = rootView.findViewById(R.id.calm_emoji);
+        angerEmojiView = rootView.findViewById(R.id.anger_emoji);
+
+        if (StatisticsFragment.parsedData == null)
+            Toast.makeText(getContext(), "No Data", Toast.LENGTH_SHORT).show();
+        else
+            initStatistics();
+        return rootView;
+    }
+
+
+    @Override
+    public void onAttach(Context context) {
+        super.onAttach(context);
+    }
+
+    private void initStatistics() {
+        DefaultFFTData defaultFFTData = new DefaultFFTData(3, 4, 2);
+        double[] data = convertToDouble(StatisticsFragment.parsedData);
+        double[][] dfftData = dataForCalculations(data);
+        defaultFFTData.updateFFTData(dfftData);
+        int[] calculatedValues = defaultFFTData.getBinRangeValues();
+
+        deltaValView.setText(String.valueOf(calculatedValues[0]));
+        thetaValView.setText(String.valueOf(calculatedValues[1]));
+        lowAlphaValView.setText(String.valueOf(calculatedValues[2]));
+        highAlphaValView.setText(String.valueOf(calculatedValues[3]));
+
+        concEmojiView.setVisibility(View.VISIBLE);
+        sleepEmojiView.setVisibility(View.VISIBLE);
+        calmEmojiView.setVisibility(View.VISIBLE);
+        angerEmojiView.setVisibility(View.VISIBLE);
+
+        if(calculatedValues[0] < 40000)
```

Comment:
Could you comment out the logic you are using here to convert scores to emotions? Also, are these alpha beta values averaged out values because the signals are emitted every sec.

Reply:
Currently these are just the bin range amounts which we are showcasing now as corresponding emotions. Have you had a look at the FFT processing functions? Was not able to get the currentFFT values sorted out. Once that is sorted, maybe we can create a custom emotion score generator function which we could use in different program modes where required. What do you say?

Reply:
@jddeep ok.